### ISSUE TITLE
Buy lightning btc with onchain btc

### ISSUE THREAD

**Issue by bobderbobs** [Reckless-Satoshi/robosats#73]
I think that this project can be a good place for a lightning btc <-> onchain btc market.
I think there will be people to offer liquidity who don't want to sell their btc.
This would also incentive people who don't want Fiat to open a channel to robosats as they can sell their lightning btc in that channel for onchain btc.

**Comment by Reckless-Satoshi** [comment]
Indeed, one possible use of the platform is as market for swaps for Lightning into other BTC forms (including on-chain)

In fact, it is already an available feature. One only has to select "BTC" (with a swap symbol) among the currencies. Then the text box will ask what is the swap destination (the order maker chooses). Additionally one could price it using the premium box (e.g. 0.3%). For example, in this screenshot, I would be creating an order to buy 100K satoshis in lightning an pay with an on-chain BTC transaction.
![swap](https://user-images.githubusercontent.com/90936742/157483521-489d3c5d-78d8-4585-ae8a-243a135aa318.png)

Possibilities are endless Lightning -> rBTC / wBTC / liquidBTC and viceversa.

Thanks for the input and ideas!
